mod-source-record-manager-server/src/test/java/org/folio/rest/impl/metadataProvider/MetadataProviderJobExecutionAPITest.java

```diff
@@ -1044,4 +1046,57 @@ private Future<JournalRecord> createJournalRecord(String jobExecutionId, String
       .withEntityHrId(entityHrid);
     return journalRecordDao.save(journalRecord, TENANT_ID).map(journalRecord);
   }
+
+  @Test
+  public void shouldNotReturnDeletedJobExecutionRecords(TestContext context) {
+    Async async = context.async();
+    JobExecution createdJobExecution = constructAndPostInitJobExecutionRqDto(1).getJobExecutions().get(0);
+    String sourceRecordId = UUID.randomUUID().toString();
+    String recordTitle = "test title";
+
+    Future<JournalRecord> future = Future.succeededFuture()
+      .compose(v -> createJournalRecord(createdJobExecution.getId(), sourceRecordId, null, null, recordTitle, 0, CREATE, MARC_BIBLIOGRAPHIC, COMPLETED, null))
+      .compose(v -> createJournalRecord(createdJobExecution.getId(), sourceRecordId, null, null, null, 0, UPDATE, MARC_BIBLIOGRAPHIC, COMPLETED, null))
```

Comment:
Hello @SatishkumarPurswani, looks like we don't need to create journal records because below in the test we don't check that they should not be returned

Reply:
Hi @RuslanLavrov - code has been updated as per your suggestions, Thank you 🙂